install/jakartaee/bin/sig-test-pkg-list.txt

```diff
@@ -37,14 +37,6 @@ jakarta.annotation
 javax.annotation.processing
 jakarta.annotation.security
 jakarta.annotation.sql
-jakarta.batch.api
-jakarta.batch.api.chunk
```

Comment:
It shouldn't matter if you keep or remove this change for signature testing as we are using docker/build_signatures.sh